File: runtimes/java/Time.java
import java.lang.reflect.*;
import java.time.temporal.TemporalAccessor;
import java.util.*;

${SOLUTION}

${UTILITIES}

public class Main {
    public static void main(String[] args) {
        long cur = System.currentTimeMillis();
        ${CALL_SOLUTION};
        long now = System.currentTimeMillis();
        System.out.println(now - cur);
    }
}

File: runtimes/kotlin/Time.kt
import java.lang.reflect.*
import java.time.temporal.TemporalAccessor
import java.util.*
import java.lang.reflect.Array

import kotlin.math.*
import java.lang.StringBuilder

${SOLUTION}

${UTILITIES}

fun main() {
    val cur = System.currentTimeMillis()
    ${CALL_SOLUTION}
    val now = System.currentTimeMillis()
    println(now - cur)
}